src/TrackGenerator.cpp

```diff
@@ -1918,14 +2024,29 @@ void TrackGenerator::initializeSegments() {
   FSRs_to_keys = _geometry->getFSRsToKeys();
 
   /* Iterate over all Track segments and assign them each a Material */
-  for (int i=0; i < _num_azim; i++) {
-    for (int j=0; j < _num_tracks[i]; j++) {
-      for (int s=0; s < _tracks[i][j].getNumSegments(); s++) {
-        curr_segment = _tracks[i][j].getSegment(s);
-	region_id = curr_segment->_region_id;
-        Material* mat = _geometry->findFSRMaterial(region_id);
-	curr_segment->_material = _geometry->findFSRMaterial(region_id);
-	FSR_keys_map->at(FSRs_to_keys->at(region_id))->_mat_id = mat->getId();
+#pragma omp parallel
```

Comment:
I'll submit a PR with multi-sim tests this evening since it's very important to me that we maintain that capability. Hopefully we can merge that and then this in within a few days. Do you plan to add tests for fsr volume correction and segment splitting? We should also have tests for track loading from a file (w/ and w/o a CMFD mesh) to ensure that tracks are always loaded when possible. Thought those tests aren't necessary here.